### PR TITLE
add error page

### PR DESCRIPTION
**Pull request by pjroggy**

### CODE REVIEW

**Review by KevinDavoust — APPROVED** [review]

File: client/src/pages/ErrorPage.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { Link } from "react-router-dom";
+import "../styles/ErrorPage404.scss";
+
+function ErrorPage404() {
+  return (
+    <section className="error404">
```
Comment (Yoann-Fortin) [inline client/src/pages/ErrorPage.jsx]: Main tag is better here

File: client/src/pages/ErrorPage.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <h2>Trou de ver E404</h2>
```
Comment (Yoann-Fortin) [inline client/src/pages/ErrorPage.jsx]: H1 is better here also

File: client/src/styles/ErrorPage404.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
+body {
+  background-image: url(../assets/images/wormhole.jpg);
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: center;
```
Comment (Yoann-Fortin) [inline client/src/styles/ErrorPage404.scss]: Dead code here

File: client/src/styles/ErrorPage404.scss [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  justify-content: center;
+  min-height: 100vh;
+}
+
+.error404 {
+  color: white;
```
Comment (Yoann-Fortin) [inline client/src/styles/ErrorPage404.scss]: A color variable is already declared but unused here

File: client/src/styles/ErrorPage404.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  box-shadow: 10px 10px 10px rgba(30, 30, 30, 0.5);
+  padding: 10px;
+  border-radius: 20px;
+}
+h2 {
+  margin: 10px;
+}
+.link {
+  text-decoration: none;
+  color: white;
```
Comment (Yoann-Fortin) [inline client/src/styles/ErrorPage404.scss]: Same

File: client/src/styles/ErrorPage404.scss [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+    rgba(255, 255, 255, 0.2),
+    rgba(255, 255, 255, 0)
+  );
+  backdrop-filter: blur(4px);
+  box-shadow: 10px 10px 10px rgba(30, 30, 30, 0.5);
+  padding: 10px;
+  border-radius: 20px;
+}
+h2 {
+  margin: 10px;
```
Comment (Yoann-Fortin) [inline client/src/styles/ErrorPage404.scss]: rem is used in this project

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

**Review by lxndrMr — APPROVED** [review]

File: client/src/App.css
Comment (Yoann-Fortin) [inline client/src/App.css:1]: No empty file please

**Review by Yoann-Fortin — CHANGES_REQUESTED** [review]

File: client/src/App.css
Comment (pjroggy) [reply client/src/App.css:1]: done !!

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Review by Yoann-Fortin — APPROVED** [review]

**Pull request merged**